x-pack/plugins/graph/server/lib/license_state.ts

```diff
@@ -7,13 +7,15 @@
 import Boom from 'boom';
 import { map } from 'rxjs/operators';
 import { Observable, Subscription } from 'rxjs';
+import { LicensingPluginStart } from '../../../licensing/server';
 import { ILicense } from '../../../licensing/common/types';
 import { checkLicense, GraphLicenseInformation } from '../../common/check_license';
 
 export class LicenseState {
   private licenseInformation: GraphLicenseInformation = checkLicense(undefined);
   private subscription: Subscription | null = null;
   private observable: Observable<GraphLicenseInformation> | null = null;
+  private _notifyUsage: LicensingPluginStart['featureUsage']['notifyUsage'] | null = null;
```

Comment:
just an thought when viewing this, why not call it `licenseNotifyUsage` to omit the `_`